recipes/flatcc/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+class FlatccConan(ConanFile):
+    name = "flatcc"
+    description = "C language binding for Flatbuffers, an efficient cross platform serialization library"
+    topics = "conan", "flatbuffers", "serialization"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dvidelabs/flatcc"
+    license = "Apache-2.0"
+    options = { "shared": [True, False],
+                "fPIC": [True, False],
+                "portable": [True, False],
+                "gnu_posix_memalign": [True, False],
+                "runtime_lib_only": [True, False],
+                "verify_assert": [True, False],
+                "verify_trace": [True, False],
+                "reflection": [True, False],
+                "native_optim": [True, False],
+                "fast_double": [True, False],
+                "ignore_const_condition": [True, False],
+    }
+    default_options = { "shared": True,
```

Comment:
```suggestion
    default_options = { "shared": False,
```

By default, any recipe is static.